.circleci/config.yml

```diff
@@ -36,23 +36,28 @@ job-references:
       - checkout
       - run: *setup_environment
       - run: *install_dependencies
+      - run:
+          name: "composer"
+          command: |
+            sudo mkdir ../../plugins
+            sudo chown circleci:circleci ../../plugins
+            composer install --dev
+            composer require "phpunit/phpunit=5.7.*"
```

Comment:
Why not place the phpunit dependencies directly in `composer.json` instead of CI config?

Reply:
Yes, that would be ideal.
I'll update the composer file

Reply:
Thanks